Client/Phpredis/Client.php

```diff
@@ -224,23 +224,23 @@ public function setnx($key, $value)
     /**
      * {@inheritdoc}
      */
-    public function del($key1, $key2 = null, $key3 = null)
+    public function del($key, ...$other_keys)
     {
         return $this->call('del', func_get_args());
```

Comment:
This actually changes the behaviour of this method when more than one parameter is passed.

When calling `del('key1', 'key2', 'key3);`

In the old situation this would translate to `$this->call('del', ['key1', 'key2', 'key3']);` but with the changed code it becomes `$this->call('del', ['key1', ['key2', 'key3']]);`

Instead `array_merge([$key], $other_keys)` should be used (or `[$key, ...$other_keys]`, but that is PHP7.4+, see https://wiki.php.net/rfc/spread_operator_for_array)

Reply:
All fixed, thanks a lot for this great catch!